plugins/ui/docs/sidebar.json

```
@@ -90,6 +90,10 @@
           {
             "label": "Update Dictionaries in State",
             "path": "add-interactivity/update-dictionaries-in-state.md"
+          },
+          {
+            "label": "Plot with deephaven.ui",
```

```suggestion
            "label": "Plotting",
```

with deephaven.ui is redundant, everything in this sidebar is deephaven.ui.

---

It doesn't matter much to me, but the reviewers from the docs team have told me multiple times that titles may not end with "-ing". So we could call it "Plot" but that sounds awkward.

---

Sounds good. I will update.